kedro_airflow_k8s/operators/create_pipeline_storage.py

```diff
@@ -0,0 +1,81 @@
+"""
+Module contains Apache Airflow operator that creates dynamic PV for data shared by
+experiment tasks.
+"""
+from typing import List, Optional
+
+from airflow.operators.python import PythonOperator
+from airflow.utils.decorators import apply_defaults
+from kubernetes import client, config
+from kubernetes.client import models as k8s
+
+
+class CreatePipelineStorageOperator(PythonOperator):
```

Comment:
There is no need to for the operators to be a subclass of PythonOperator, consider using BaseOperator instead (it's how other Airflow operators do)

Reply:
Thanks for the hint, will do that.